tests/phase3_provider_acquisition_test.sh

```
@@ -0,0 +1,196 @@
+    "https://releases.grapheneos.org/shiba-ota_update-2026071700.zip" \
+    "${GRAPHENEOS[OTA_URL]}" "GrapheneOS OTA URL"
+  assert_equals "v29.0" "${VERSION[MAGISK]}" "Magisk version"
+  assert_equals "" "${ROM_OTA_SHA256}" "GrapheneOS digest availability"
+)
+
+test_lineageos_v2_metadata() (
+  reset_acquisition_fixture
+  ROM_FAMILY="lineageos"
+  DEVICE_NAME="pdx235"
+
+  git() { mock_magisk_tags "$@"; }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)